site/src/pages/components/date-picker/index.mdx

```diff
@@ -10,6 +10,8 @@ Date components allow the user to make a specific selection through a visual cal
 - The user should be able to enter values also through the keyboard.
 - Use placeholders to indicate the format of the data entry, for example: dd/mm/yyyy or mm/dd/yy, etc..
 - The field should have only one input type, and should not be divided into fields for each information (day, month, year).
+- Use the Week Range Picker if the selected periods must start on a Sunday and end on a Saturday.
+- If you place a base value on the Week Range Picker, it must be a week, for example: 05/02/2021 (Sunday) to 05/08/2021 (Saturday). Otherwise, it will not be selected. An example of a invalid entry: 05/03/2021 (Monday) to 05/05/2021 (Wednesday).
```

Comment:
what do you think of something like this: `If the Week Range Picker receive a value from a prop, it must ...`? Idk, I think `base value` sounds a bit weird heheh :sweat_smile:

Reply:
Much better hehe